src/test/java/org/tarantool/jdbc/AbstractJdbcIT.java

```diff
@@ -36,8 +36,9 @@
         "CREATE TABLE test(id INT PRIMARY KEY, val VARCHAR(100))",
         "INSERT INTO test VALUES (1, 'one'), (2, 'two'), (3, 'three')",
         "CREATE TABLE test_compound(id1 INT, id2 INT, val VARCHAR(100), PRIMARY KEY (id2, id1))",
-        "CREATE TABLE test_nulls(id INT PRIMARY KEY, val VARCHAR(100))",
-        "INSERT INTO test_nulls VALUES (1, 'a'), (2, 'b'), (3, 'c'), (4, NULL), (5, NULL), (6, NULL)",
+        "CREATE TABLE test_nulls(id INT PRIMARY KEY, val VARCHAR(100), dig INTEGER)",
+        "INSERT INTO test_nulls VALUES (1, 'a', 10), (2, 'b', 20), (3, 'c', 30), " +
+            "(4, NULL, NULL), (5, NULL, NULL), (6, NULL, NULL)",
```

Comment:
Not about your specific commit, but I think about having a test initialization and test cases in one file again and again. It is inconvenient to jump over several files to look at a simple test case. What do you think?

Reply:
Agreed. We need to refactor and change our approach to writing readable tests. We can start to increase test locality moving _arrange-act-assert_ pattern within one test method or at most in one test class for the reused arrangement (@before/@after). It's better to file it as a separate issue, I think.

Reply:
Okay, let's do.

Reply:
For the record: https://github.com/tarantool/tarantool-java/issues/188